app/config/config_dev.yml

```diff
@@ -15,7 +15,7 @@ monolog:
     handlers:
         main:
             type: stream
-            path: "%kernel.logs_dir%/%kernel.environment%.log"
+            path:   "/tmp/stdout"
```

Comment:
Sorry about that. There is a problem with Docker and the stdout stream that will not allow non-root users (like, www-data) from writing to it.  I've added an `entrypoint.sh` to handle this with more documentation.